spec/unit/run_spec.rb

```diff
@@ -160,4 +160,15 @@
       "[\e[32m#{uuid}\e[0m] Finished in x seconds with exit status 0 (\e[32;1msuccessful\e[0m)\n"
     ])
   end
+
+  it "does not persist environment variables" do
+    output = StringIO.new
+    command = TTY::Command.new(output: output)
+
+    out, _err = command.run("env | grep FOO", env: {foo: 1})
+    expect(out.chomp).to eq("FOO=1")
+    out, _err = command.run!("env | grep FOO")
+    expect(out).to eq("")
+  end
```

Comment:
I'd consider examining the `output` format as it will show the commands that are run with the `export` statement. Also, it would be good to get this running on Windows which probably won't have `grep` utility. I tend to stay with the safer `echo` command.

Reply:
Sure, updating.

Reply:
I updated the tests, borrowing from a similar test in the same spec.

If it was my code, I would have probably opted to use something like:

```ruby
expect(output.string).to include "..."
```
instead of all the `readlines` acrobatics, but since your gems need to work on more environments, I figured I play it safe since you must have a reason for doing it this way.

---

As a side note, the styling bot is super annoying - if it was a real robot, I would have disassembled it and sold it for parts... 😏

Reply:
I'm ok with any approach that verifies behaviour and passes tests. I updated all the test setup to work on GitHub Actions and Appveyor so you should get immediate feedback. The style bot is annoying but it has good intentions...😉